src/ktopt_planning_context.cpp

```diff
@@ -248,29 +253,84 @@ void KTOptPlanningContext::transcribePlanningScene(const planning_scene::Plannin
     }
     if (object == OCTOMAP_NS)
     {
-      RCLCPP_ERROR(getLogger(), "skipping octomap for now ...");
+      RCLCPP_INFO(getLogger(), "skipping octomap for now ...");
       continue;
     }
+    RCLCPP_INFO(getLogger(), "iterating inside collision object's shapes");
     for (const auto& shape : collision_object->shapes_)
     {
-      RCLCPP_ERROR(getLogger(), "iterating inside collision object's shapes");
       const auto& pose = collision_object->shape_poses_[0];
+      const auto& shape_type = collision_object->shapes_[0]->type;
+      RCLCPP_INFO(getLogger(), "Shape type: %d, object id: %s, shape: %s", shape_type, object, shape);
+
+      switch (shape_type){
+        case shapes::ShapeType::BOX: {
+
+          const auto objectptr = std::dynamic_pointer_cast<const shapes::Box>(shape);
+          RCLCPP_INFO(getLogger(), "Box, size: %f", objectptr->size[0]); // shape.size
+          const SourceId box_source_id = scene_graph.RegisterSource("template_box");
```

Comment:
The name should probably not be "template_SHAPE" but rather the actual entity name from the planning scene?